x-pack/legacy/plugins/siem/public/components/page/network/network_dns_table/index.tsx

```diff
@@ -93,7 +113,7 @@ export const NetworkDnsTableComponent = React.memo<NetworkDnsTableProps>(
           }
         }
       },
-      [sort, type]
+      [sort, type, tableType]
```

Comment:
Why is `tableType` necessary here but not on `updateLimitPagination` & `updateActivePage`? I see it's excluded for most other callbacks as well.

Seems to be differing usages between here and the other tables `onChange()` as well -- I might be missing something here, but just want to verify what difference changes the consistency between tables.

https://github.com/elastic/kibana/blob/1fa6e33f85607ee78ed792ca9f9a34d6bcaa27ba/x-pack/legacy/plugins/siem/public/components/page/network/network_dns_table/index.tsx#L116

https://github.com/elastic/kibana/blob/1fa6e33f85607ee78ed792ca9f9a34d6bcaa27ba/x-pack/legacy/plugins/siem/public/components/page/network/network_http_table/index.tsx#L118

https://github.com/elastic/kibana/blob/1fa6e33f85607ee78ed792ca9f9a34d6bcaa27ba/x-pack/legacy/plugins/siem/public/components/page/network/network_top_countries_table/index.tsx#L157

https://github.com/elastic/kibana/blob/1fa6e33f85607ee78ed792ca9f9a34d6bcaa27ba/x-pack/legacy/plugins/siem/public/components/page/network/network_top_n_flow_table/index.tsx#L132

Reply:
It was my mistake, the first round was done without any help, so I've added `tableType` to dependencies even though it was unnecessary, so to make sure that all dependencies are correct I've enabled temporarly `react-hooks/exhaustive-deps` rule and fixed everything as it suggested. I believe now it should be correct